main.js

```diff
@@ -0,0 +1,237 @@
+    paragraphElement.textContent = programs[index].description;
+
+    // Append H2, paragraph, and close button to popup content
+    popupContent.appendChild(closeButton);
+    popupContent.appendChild(h2Element);
+    popupContent.appendChild(paragraphElement);
+
+    // Append popup content to popup container
+    popupContainer.appendChild(popupContent);
+
+    // Append popup container to the body
+    document.body.appendChild(popupContainer);
+
+    // Add click event listener to close the popup
+    closeButton.addEventListener('click', () => {
+      // Remove the popup container
+      popupContainer.remove();
+    });
+  });
+});
+
+// person section
+
+const persons = [
+  {
+    name: 'Peter',
+    image: 'images/peter2.jpg',
+    description: 'Peter, our accomplished CEO, leads with vision and expertise, propelling our company to new heights of success. His strategic acumen and commitment to innovation drive our team to deliver exceptional results.',
+    role: 'Chief Executive Officer (CEO)',
+  },
+  {
+    name: 'Mayssa',
+    image: 'images/mayssa5.webp',
+    description: "Responsible for developing and executing the company's marketing strategies.",
+    role: 'Chief Marketing Officer (CMO)',
+  },
+  {
+    name: 'Santiago',
+    image: 'images/santiago3.jpeg',
+    description: "Manages the company's financial operations, including budgeting, financial planning, and financial reporting.",
+    role: 'Chief Financial Officer (CFO)',
+  },
+  {
+    name: 'Catherine',
+    image: 'images/catherine4.jpeg',
+    description: 'Oversees the day-to-day operations of the company, ensuring efficient and effective processes across various departments.',
+    role: 'Chief Operations Officer (COO)',
+  },
+  {
+    name: 'Stellar',
+    image: 'images/stellar1.jpeg',
+    description: 'Responsible for aligning technology initiatives with business objectives, managing IT infrastructure, and driving digital transformation.',
+    role: 'Chief Technology Officer (CTO)',
+  },
+  {
+    name: 'Houda',
+    image: 'images/houda6.jpg',
+    description: 'Oversees all aspects of human resources, including recruitment, employee relations, training and development, and HR policies.',
+    role: 'Human Resources Director',
+  },
+];
+
+const section3 = document.querySelector('.section3');
+
+let displayedPersons = persons.slice(0, 5);
+
+function renderPersons(personsArray) {
+  const htmlContent = `
+    <h2>Founders</h2>
+    <div class="line"></div>
+    <div class="speakers">
+      ${personsArray
+    .map(
+      (person) => `
+          <div class="person">
+            <img class="personImage" src="${person.image}" alt="${person.name}_image">
+            <div class="clm2">
+              <h2 class="name">${person.name}</h2>
+              <p class="p1">${person.description}</p>
+              <div class="line2"></div>
+              <p class="p2">${person.role}</p>
+            </div>
+          </div>
+        `,
+    )
```

Comment:
- [ ] @Nessrine88 Great work implementing the changes requested, however, while doing it you removed one speaker, I have to go to the mobile menu and click more to have all speakers in the desktop view , kindly consider fixing the error
![image](https://github.com/Nessrine88/Capstone-Project/assets/61408860/f2df92d3-f09d-4433-9e64-82e11d544b8b)